src/tests/lxc-test-apparmor-mount

```diff
@@ -158,6 +158,9 @@ if [ -f /etc/lsb-release ]; then
 fi
 
 run_cmd lxc-create -t download -n $cname -- -d ubuntu -r $release -a $ARCH
+echo "lxc.include =" >> $HDIR/.local/share/lxc/$cname/config
+echo "lxc.include = /usr/share/lxc/config/common.conf" >> $HDIR/.local/share/lxc/$cname/config
+echo "lxc.include = /usr/share/lxc/config/userns.conf" >> $HDIR/.local/share/lxc/$cname/config
```

Comment:
We should drop that particular commit, we'll have to make sure our images don't have that issue before we merge it.